### PR TITLE
feat(observability): migrate dashboard from grabana to grafana foundation sdk

### PR DESCRIPTION
**Pull request by Atrax1**
This PR brings the following : 
- CLI with deploy and generate commands
- Can deploy dashboards to grafana instance
- Can generate grafana JSON dashboard that can be imported to a grafana instance
- Available dashboards: Core Node Component Health, Core Node General Dashboard, Core Node Resources Usage, DON OCR1, OCR2 and OCR3 Dashboards

### CODE REVIEW

File: observability-lib/go.mod [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+go 1.21.4
+
+require (
+	github.com/K-Phoen/grabana v0.22.1
+	github.com/grafana/grafana-foundation-sdk/go v0.0.0-20240415154618-2fd5b8b03249
+	github.com/rs/zerolog v1.32.0
+	github.com/smartcontractkit/chainlink-testing-framework/grafana v0.0.0-20240508164646-7ae47c88ecbb
+	github.com/spf13/cobra v1.8.0
+	github.com/stretchr/testify v1.8.4
```
Comment (jmank88) [inline observability-lib/go.mod:11]: 1.9.0 was released in march :shrug:

**Review by jmank88 — APPROVED** [review]

File: observability-lib/go.mod (hunk 1 above)
Comment (Atrax1) [reply observability-lib/go.mod]: right just upgraded it

File: observability-lib/main.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+func init() {
+	rootCmd.AddCommand(cmd.DeployCmd)
+	rootCmd.AddCommand(cmd.GenerateCmd)
+}
+
+func main() {
+	if err := rootCmd.Execute(); err != nil {
+		log.Println(err)
+		os.Exit(1)
```
Comment (jmank88) [inline observability-lib/main.go:25]: nit/ is this the same?
```suggestion
		log.Fatalln(err)
```

**Review by jmank88 — APPROVED** [review]

File: observability-lib/main.go (hunk 2 above)
Comment (Atrax1) [reply observability-lib/main.go]: thanks for the suggest

**Review by jmank88 — APPROVED** [review]

**Review by patrickhuie19 — APPROVED** [review]

**Pull request merged**